lib/index.js

```diff
@@ -258,8 +270,20 @@ Appboy.prototype.track = function(track) {
     delete properties[key];
   }, reserved);
 
-  window.appboy.changeUser(userId);
-  window.appboy.logCustomEvent(eventName, properties);
+  if (this.supportsMessagingReadyCallback) {
```

Comment:
@ladanazita @ucarion @viparthasarathy should the same userId code from `prototype.page` go here as well?

Reply:
Yes, it should.

Reply:
Gotcha, will update. I'm questioning if changeUser is even really needed here (pretty sure it's always a no-op because either it was cached and we've called changeUser at initialization time or we've called identify and called changeUser in there), but don't want to get sidetracked.